src/VisionModel/AlexNet.jl

```diff
@@ -0,0 +1,25 @@
+
+function AlexNet():
+  layers = Chain(Conv((11, 11), 3=>64, stride=(4, 4), relu, pad=(2, 2)),
+            MaxPool((3, 3), stride=(2, 2)),
+            Conv((5, 5), 64=>192, relu, pad=(2, 2)),
+            MaxPool((3, 3), stride=(2, 2)),
+            Conv((3, 3), 192=>384, relu, pad=(1, 1)),
+            Conv((3, 3), 384=>256, relu, pad=(1, 1)),
+            Conv((3, 3), 256=>256, relu, pad=(1, 1)),
+            MaxPool((3, 3), stride=(2, 2)),
+            AdaptiveAvgPool((6,6)),
+            x -> flatten(x, 1),
+            Dropout(),
+            Dense(256 * 6 * 6, 4096, relu),
+            Dropout(),
+            Dense(4096, 4096, relu),
+            Dense(4096, 1000)
+            )
+Flux.testmode!(layers)
+return layers
+end
+end
+functor(AlexNet)
+Base.show(io::IO, ::AlexNet) = print(io, "AlexNet()")
```

Comment:
Remember, this won’t work because `AlexNet` is no longer a type.

Reply:
I have not used functor as well as that ``.show()`` function.